.travis.yml

```diff
@@ -17,25 +19,19 @@ notifications:
     on_success: change
     on_failure: change
 
-# System dependencies for HTTP calling
-#r_binary_packages:
-# - dplyr
-
-r_packages:
-  - devtools
-  - Rcpp
+r_github_packages:
+  - hadley/pkgdown
+  - andrewpbray/infer
```

Comment:
@andrewpbray It is a little weird to have to install the package here via GitHub and this is a slight concern. `pkgdown` won't build the site without installing the package though?